textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/utils/DataflowUtils.java

```diff
@@ -294,4 +295,25 @@ public static String getSpanString(Span span) {
         return payload;
     }
 
+    public static JSONArray getMetadataJSON(List<TableMetadata> list) {

```

Comment:
I introduced a new way to automatically serialize the `Schema` class to JSON using the Jackson library. Can you change it to use that way instead? 
For example, if you have a `schema` object, you can simply do `new ObjectMapper().writeValueAsString(schema)` and you have the JSON string of the schema.